WordPress/Sniffs/Theme/DeregisterScriptSniff.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
```

Comment:
What about naming the sniff `NoDeregisterScript` to make it more obvious what it does ?